sdk/compiler/daml-extension/src/language_client.ts

```diff
@@ -0,0 +1,277 @@
+    baseArgs: string[],
+    toAdd: [string, string[]][],
+  ): string[] {
+    let addedArgs: string[][] = toAdd
+      .filter(x => config.get(x[0]))
+      .map(x => x[1]);
+    addedArgs.unshift(baseArgs);
+    return [].concat.apply([], <any>addedArgs);
+  }
+
+  private getLanguageServerArgs(
+    config: vscode.WorkspaceConfiguration,
+    telemetryConsent: boolean | undefined,
+    identifier: string,
+  ): string[] {
+    const multiIDESupport = config.get("multiPackageIdeSupport");
+    this.isMultiIde = !!multiIDESupport;
+    const logLevel = config.get("logLevel");
+    const isDebug = logLevel == "Debug" || logLevel == "Telemetry";
+
+    let args: string[] = [multiIDESupport ? "multi-ide" : "ide", "--"];
+
+    if (telemetryConsent === true) {
+      args.push("--telemetry");
+    } else if (telemetryConsent === false) {
+      args.push("--optOutTelemetry");
+    } else if (telemetryConsent == undefined) {
+      // The user has not made an explicit choice.
+      args.push("--telemetry-ignored");
+    }
+    if (multiIDESupport === true) {
+      args.push("--log-level=" + logLevel);
+      args.push("--ide-identifier=" + identifier);
+    } else {
+      if (isDebug) args.push("--debug");
+    }
+    const extraArgsString = config.get("extraArguments", "").trim();
+    // split on an empty string returns an array with a single empty string
+    const extraArgs = extraArgsString === "" ? [] : extraArgsString.split(" ");
+    args = args.concat(extraArgs);
+    const serverArgs: string[] = this.addIfInConfig(config, args, [
+      ["profile", ["+RTS", "-h", "-RTS"]],
+      ["autorunAllTests", ["--studio-auto-run-all-scenarios=yes"]],
+    ]);
+
+    return serverArgs;
+  }
+
+  // Update the manual watchers in haskell side to be aware of the root directory, and only ask for that
+
+  private createLanguageClient(
+    rootPath: string,
+    envVars: { [envVarName: string]: string },
+    config: vscode.WorkspaceConfiguration,
+    telemetryConsent: boolean | undefined,
+    identifier: string,
+  ): LanguageClient {
+    // Options to control the language client
+    let clientOptions: LanguageClientOptions = {
+      // Register the server for Daml
+      documentSelector: [
+        { language: "daml", pattern: rootPath + "/**/*.daml" },
+      ],
+    };
+
+    let command: string;
+
+    try {
+      command = which.sync("daml");
+    } catch (ex) {
+      const damlCmdPath = path.join(damlRoot, "bin", "daml");
+      if (fs.existsSync(damlCmdPath)) {
+        command = damlCmdPath;
+      } else {
+        vscode.window.showErrorMessage(
+          "Failed to start the Daml language server. Make sure the assistant is installed.",
+        );
+        throw new Error("Failed to locate assistant.");
+      }
+    }
+
+    const serverArgs = this.getLanguageServerArgs(
+      config,
+      telemetryConsent,
+      identifier,
+    );
+    console.log(serverArgs);
+    console.log(envVars);
+
```

Comment:
Previously was setting envVars["DAML_SDK_VERSION"] to "0.0.0", as theres no `daml.yaml` where the multi-ide starts, and it was defaulting to highest installed version (i.e. not 0.0.0)
Once this is released, this should fix it, but for now, you need to ensure you _only_ have 0.0.0 installed to test it